src/rust/e2e-tests/src/test_utils/plugin_health.rs

```diff
@@ -0,0 +1,41 @@
+use std::time::Duration;
+
+use rust_proto::graplinc::grapl::api::plugin_registry::v1beta1::{
+    GetPluginHealthRequest,
+    PluginHealthStatus,
+    PluginRegistryClient,
+};
+
+/// After a deploy, plugins seem to take quite some time to deploy.
```

Comment:
Should this be "After a deploy, plugins seem to take quite some time to become healthy"?

Reply:
fixed